src/v0/util/index.js

```diff
@@ -1640,7 +1640,7 @@ const handleRtTfSingleEventError = (input, error, reqMetadata) => {
     ...getEventReqMetadata(input)
   });
 
-  return resp;
+  return { ...resp, destination: input.destination };
```

Comment:
If this is definitely needed, we might have to update all the route transform destinations that don't use this method to handle the event-level errors.

Reply:
Added the information to wherever it is needed